website/docs/docs/supported-data-platforms.md

```diff
@@ -15,17 +15,17 @@ With a few exceptions [^1], all adapters listed below can be installed from PyPI
 
 ### Verified by dbt Labs
 
-In order to provide a more consistent and reliable experience, dbt Labs now has a rigorous process by which we verify adapter plugins. The process covers aspects of development, documentation, user experience and maintenance. These adapters earn a "Verified" designation so that users can have a certain level of trust and expectation when they use them. To learn more see ["Verifying a new adapter](7-verifying-a-new-adapter")
+In order to provide a more consistent and reliable experience, dbt Labs now has a rigorous process by which we verify adapter plugins. The process covers aspects of development, documentation, user experience and maintenance. These adapters earn a "Verified" designation so that users can have a certain level of trust and expectation when they use them. To learn more see [Verifying a new adapter](docs/contributing/adapter-development/7-verifying-a-new-adapter)
```

Comment:
is it necessary and/or convention to use the full slug? I thought if the filename is unique it is sufficient to just list the doc id.
```suggestion
In order to provide a more consistent and reliable experience, dbt Labs now has a rigorous process by which we verify adapter plugins. The process covers aspects of development, documentation, user experience and maintenance. These adapters earn a "Verified" designation so that users can have a certain level of trust and expectation when they use them. To learn more see [Verifying a new adapter](7-verifying-a-new-adapter)
```

Reply:
that's a great question and i _believe_ it's better to write the full slug for the purposes of redirects and avoiding duplications. I'll loop in @nghi-ly on this but we just spoke about how difficult it is, maintenance-wise, if only the filename is used. 

I'll look at this tomorrow as well! 🤗

Reply:
yes please include the Docusaurus basepath! it makes updating the `_redirects` file way easier without creating broken links. 

please also include a leading forward slash (/). 

A perfect example of linking to an internal link is: https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/docs/guides/getting-started/building-your-first-project/schedule-a-job.md?plain=1#L55

thanks!

Reply:
sounds like another good addition to our style guide!